Source/WebCore/css/CSSProperties.json

```diff
@@ -9237,6 +9217,26 @@
                 "url": "https://www.w3.org/TR/css-backgrounds-3/#background-image"
             }
         },
+        "<single-background-repeat>": {
+            "grammar": "<repeat-style>",
+            "exported": true,
+            "specification": {
+                "category": "css-backgrounds",
+                "url": "https://www.w3.org/TR/css-backgrounds-3/#background-repeat"
+            }
+        },
+        "<single-background-position-x>": {
+            "grammar": "<position-x>",
+            "exported": true,
+            "status": "non-standard",
+            "comment": "Internally used to represent horizontal part of standard <bg-position>"
+        },
+        "<single-background-position-y>": {
+            "grammar": "<position-y>",
+            "exported": true,
+            "status": "non-standard",
+            "comment": "Internally used to represent vertical part of standard <bg-position>"
+        },
```

Comment:
Can we define `<repeat-style>`/`<position-x>`/`<position-y>` from `CSSProperties.json` as well?

Reply:
Not at the moment. No.